Object object
Rect object_bbox
Location3D location
